subsys/http/src/main/conf/http.conf

```diff
@@ -0,0 +1,54 @@
+[http]
+## Enable http config. Default is true
+#enabled=true
```

Comment:
It's hard for me to imagine how Indy could function without this, so I'd be okay taking out the enabled flag we usually use in Indy configuration classes. WDYT?